src/patternfly/demos/HelperText/examples/HelperText.md

```diff
@@ -0,0 +1,96 @@
+      {{#> form-control controlType="input" input="true" form-control--attribute=(concat 'required type="text" id="' form--id form-group--id '" name="' form--id form-group--id '" aria-invalid="true" aria-describedby="' form--id form-group--id '-helper"')}}
+      {{/form-control}}
+      {{#> form-helper-text form-helper-text--type="div" form-helper-text--modifier="pf-m-error" form-helper-text--attribute=(concat 'id="' form--id form-group--id '-helper" aria-live="polite"')}}
+          {{#> helper-text helper-text--type="ul" helper-text-item--type="li"}}
+            {{#> helper-text-item helper-text-item--modifier="pf-m-dynamic pf-m-success"}}
+              {{> helper-text-item-icon helper-text-item-icon--type="check"}}
+              {{#> helper-text-item-text}}This criteria has been met.{{/helper-text-item-text}}
+            {{/helper-text-item}}
+            {{#> helper-text-item helper-text-item--modifier="pf-m-dynamic pf-m-error"}}
+              {{> helper-text-item-icon helper-text-item-icon--type="times"}}
+              {{#> helper-text-item-text}}This criteria has not been met.{{/helper-text-item-text}}
+            {{/helper-text-item}}
+            {{#> helper-text-item helper-text-item--modifier="pf-m-dynamic pf-m-success"}}
+              {{> helper-text-item-icon helper-text-item-icon--type="check"}}
+              {{#> helper-text-item-text}}This criteria has been met.{{/helper-text-item-text}}
+            {{/helper-text-item}}
+          {{/helper-text}}
+      {{/form-helper-text}}
+    {{/form-group-control}}
+  {{/form-group}}
+  {{#> form-group form-group--id="-comment"}}
+    {{#> form-group-label}}
+      {{#> form-label form-label--attribute=(concat 'for="' form--id form-group--id '"')}}
+        Comment
+      {{/form-label}}
+    {{/form-group-label}}
+    {{#> form-group-control}}
+      {{#> form-control controlType="input" input="true" form-control--modifier="pf-m-success" form-control--attribute=(concat 'value="This is a valid comment"' 'type="text" id="' form--id form-group--id '" name="' form--id form-group--id '" aria-describedby="' form--id form-group--id '-helper"')}}
+      {{/form-control}}
+      {{#> form-helper-text form-helper-text--type="div" form-helper-text--modifier="pf-m-success" form-helper-text--attribute=(concat 'id="' form--id form-group--id '-helper" aria-live="polite"')}}
+          {{#> helper-text}}
+            {{#> helper-text-item helper-text-item--modifier="pf-m-dynamic pf-m-success"}}
+              {{> helper-text-item-icon helper-text-item-icon--type="check"}}
+              {{#> helper-text-item-text}}This is dynamic helper text with an icon showing success.{{/helper-text-item-text}}
+            {{/helper-text-item}}
+          {{/helper-text}}
+      {{/form-helper-text}}
+    {{/form-group-control}}
+  {{/form-group}}
+{{/form}}
+```
+## Documentation
+When adding helper text to a form field, place it within the form control and form helper text to ensure proper spacing.
```

Comment:
Not sure if you need this, the demo shows the structure - but if you keep it, maybe say something like this? "place it within the form control" is kinda confusing since the form control would likely refer to the text input/textarea/select/etc.

```suggestion
When adding helper text to a form field, place it in the form helper text element.